package.json

```diff
@@ -1,53 +1,34 @@
 {
   "name": "@sajari/sdk-node",
-  "version": "2.0.0-beta.7",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "author": "Ben Hinchley <bhinchley@sajari.com>",
+  "version": "4.0.0-alpha.1",
+  "repository": "git@github.com:github.com/sajari/sdk-node.git",
+  "author": "Sajari Pty Ltd",
   "license": "MIT",
+  "main": "./build/src/index.js",
+  "types": "./build/src/index.d.ts",
   "files": [
     "README.md",
     "LICENSE",
-    "dist/",
-    "generated/"
+    "build/src",
+    "!build/src/**/*.map"
   ],
-  "engines": {
-    "node": ">= 10.0.0"
-  },
-  "scripts": {
-    "prebuild": "npm-run-all gen:useragent lint test",
-    "build": "tsc",
-    "fmt": "prettier --write --config .prettierrc.js \"{src/**/*,*.config,*.d,.*,package}.{js,ts,json}\"",
-    "lint": "tslint --fix -p tsconfig.json",
-    "type-check": "tsc --noEmit",
-    "pretest": "npm-run-all type-check",
-    "test": "jest",
-    "test:watch": "jest --watch",
-    "coverage": "jest --coverage",
-    "gen:proto": "node scripts/genproto",
-    "gen:useragent": "node scripts/genuseragent",
-    "docs": "typedoc ./src/index.ts --mode file --out ./docs --exclude \"**/*+(.test).ts\"",
-    "prepublishOnly": "npm-run-all build"
-  },
   "dependencies": {
-    "debug": "^4.1.1",
-    "deepmerge": "^4.0.0",
-    "grpc": "^1.22.2",
-    "protobufjs": "^6.8.8"
+    "request": "2.88.2",
```

Comment:
Any concerns with this being deprecated? 
https://www.npmjs.com/package/request
https://github.com/request/request/issues/3142

Reply:
Unfortunately the Node SDK is generated at this stage and it uses `request`. There's no reason we cannot change it later though—it's hopefully an internal detail that doesn't leak to users.

The other option is that we will just write our own template for the generator. Or modify the existing one.